conf/modules.config

```diff
@@ -162,6 +164,115 @@ process {
         publishDir = [ enabled: false ]
     }
 
+    withName: '.*TAXID_BAM_FASTA_SHORTREAD:SAMTOOLS_IDXSTATS' {
+        publishDir = [ enabled : false ]
+    }
+
+    withName: '.*TAXID_BAM_FASTA_SHORTREAD:SUBSET_BAM_PASS' {
+        ext.prefix = { "${meta.id}_${meta.taxid}"}
+        ext.args = '-bh'
```

Comment:
What does this argument do?

Reply:
It ensures the output bam files contains header. I have tested again without this argument and it worked. So I will delete this argument.

Reply:
Maybe, it is good to have it?

Reply:
Ok. Then I will keep it.